src/js/referrals.form.js

```diff
@@ -0,0 +1,34 @@
+$('form[name="form-referrals"]').validate({
+  // required fields
+  nameReferrals: 'required',
+  phoneReferrals: 'required',
+  messageReferrals: 'required',
+  mailReferrals: {
+    required: true,
+    // Specify that email should be validated
+    // by the built-in "email" rule
+    email: true
+  },
+  messages: {
+    emailCareers: "Por favor introduce un correo valido."
+  },
+  submitHandler: function(form) {
+    var data = $('form[name="form-referrals"]').serialize();
+    // $.ajax({
```

Comment:
If you dont use this part of code, remove it

Reply:
will be used once the views in integration